README.md

```diff
@@ -24,101 +24,78 @@ Reliable decentralized exchange (DEX) blockchain - interoperable with other bloc
 The design of the blockchain guarantees fixed-fees that provides greater control of trading costs and higher arbitrage opportunity.
 Assets on the exchange will serve multiple purposes- at the first iteration, they are the block producer’s stake and exchange liquidity at the same time, and more comes later.
 
-## Local Development
+## Build mangata-node locally
+- Install [docker](https://docs.docker.com/engine/install/ubuntu/) 
 
-Follow these steps to prepare a local Substrate development environment :hammer_and_wrench:
+### Compilie mangata-node binary and wasms artifacts
+- use docker wrapper for cargo to build `mangata-node`
 
-### Simple Setup
-
-Install all the required dependencies with a single command (be patient, this can take up to 30
-minutes).
-
-```bash
-curl https://getsubstrate.io -sSf | bash -s -- --fast
+```
+./docker-cargo.sh build --release -p mangata-node
 ```
 
-### Manual Setup
-
-Find manual setup instructions at the
-[Substrate Developer Hub](https://substrate.dev/docs/en/knowledgebase/getting-started/#manual-installation).
-
-## Build
-### Local target
-
-Recommended rustc version for the build is `nightly-2021-10-19`
+build artifacts will be placed in `<REPO ROOT>/docker-cargo/release`
 
-Environment variables for ethereum apps should be set up before the build:
+### Generate docker image
+You can use `build-image.sh` script to build & generate docker image
 
-```bash
-ETH_APP_ID=0xdd514baa317bf095ddba2c0a847765feb389c6a0
-ERC20_APP_ID=0x00e392c04743359e39f00cd268a5390d27ef6b44
 ```
-
-build node:
-
-```bash
-rustup target add wasm32-unknown-unknown
-cargo build --release
+./scripts/build-image.sh
 ```
 
-### Docker container
+of you can use already compiled build atributes generated in previous step
 
-```bash
-./scripts/build-mangata-node-docker-image.sh
+```
+SKIP_BUILD=1 BUILD_DIR=./docker-cargo/release ./scripts/build-image.sh
 ```
 
-## Run
-
-### Single Node Development Chain
-
-Purge any existing dev chain state:
+This will generate new local docker image `mangatasolutions/mangata-node:dev`
 
-```bash
-./target/release/mangata-node purge-chain --dev
-```
+## Run
 
-Start a dev chain:
+In order to run mangata-parachain locally one need to set up both:
+- local relay network
+- local parachain network
 
-```bash
-./target/release/mangata-node --dev
-```
+Because of number of parameters is quite troublesome thats why we came up with dedicated dockerized environment.
 
-### Two-Nodes Dockerized Testnet
+### Set up network using parachain-launch
 
-```bash
-cd ./devops
-docker-compose up
-```
+Dockerized setup requires you to build development docker image [mangatasolutions/mangata-node:dev](#generate-docker-image).
 
-### Two-Nodes Multi-Validator Dockerized Testnet
+Start docker environment using, you need to 
 
 ```bash
-sh scripts/build-multi-validator-mangata-node-docker-image.sh
-docker-compose  -f devops/multi-validator-docker-compose.yml up
+cd ./launch
+yarn install
+yarn gen
+yarn up
```

Comment:
can someone validate these steps? recently i had to call instead of the `yarn gen`
`node_modules/.bin/parachain-launch generate`